B827EBFFFE553592.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE553592",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```

Comment:
This is the old V2 router, you need to update to eu1.cloud.thethings.network

Reply:
I've changed it. Cold you please check it? Do I need to make something else? Thanks in advance!

Reply:
Done and merged!